api/v1alpha1/eacruntime_types.go

```diff
@@ -142,7 +142,7 @@ type EACRuntimeSpec struct {
 	Replicas int32 `json:"replicas,omitempty"`
 
 	// Operation system optimization for EAC
-	OperationSystemOptimization OperationSystemOptimization `json:"operationSystemOptimization,omitempty"`
+	OSAdvise OperationSystemOptimization `json:"osAdvise,omitempty"`
```

Comment:
Can you rename OperationSystemOptimization to OSAdvise?

Reply:
done

Reply:
> Can you rename OperationSystemOptimization to OSAdvise?

done